pkg/defaults/deployment_defaults.go

```diff
@@ -29,13 +29,11 @@ const (
 	// GroupName is the group name for istio labels and annotations
 	GroupName = "service.istio.io"
 
-	// RevisionLabelKey is the label key attached to k8s resources to indicate
-	// which Revision triggered their creation.
+	// RevisionLabelKey is the label key to define the canonical revision name used by istio.
 	RevisionLabelKey = GroupName + "/canonical-revision"
 
-	// ServiceLabelKey is the label key attached to a Route and Configuration indicating by
-	// which Service they are created.
-	ServiceLabelKey = GroupName + "/canonical-service"
+	// ServiceLabelKey is the label key to define the canonical service name used by istio.
```

Comment:
would this be a backward compatible change?

Reply:
No, the old was was just wrong.